src/main/java/org/folio/converter/Field008Items.java

```diff
@@ -0,0 +1,74 @@
+package org.folio.converter;
+
+public enum Field008Items {
```

Comment:
Is it possible to rename Field008 to smth like FixedLengthControlField (https://www.loc.gov/marc/bibliographic/bd00x.html) in all the places of converter?

Reply:
Done